spec/public/versions_spec.rb

```diff
@@ -42,4 +42,24 @@
       expect(subject.find!('stable').to_s).to eq('2024-04-01')
     end
   end
+
+  describe '#find_next' do
```

Comment:
can we add a case / context with unsorted versions? it looks like the original array is already sorted